Breeze.UI/src/app/wallet/tumblebit/password-confirmation/password-confirmation.component.html

```diff
@@ -8,7 +8,15 @@
         <h5 class="modal-title text-uppercase" id="modalReceive" *ngIf="!startingTumble">Password Confirmation</h5>
         <form class="col-10 mt-4 offset-1" [formGroup]="walletPasswordForm" *ngIf="!startingTumble">
           <div class="form-group clearfix">
-            <p class="text-muted text-center"><em>Please enter your source wallet password.</em></p>
+            <p class="text-muted text-center">
+              <em>By starting the Breeze Privacy Protocol you are subjecting your entire balance {{balance | coinNotation}}BTC to be shuffled by the Breeze Privacy Protocol.</em>
```

Comment:
Rather than BTC should use `{{CoinUnit}}` like in Dashboard

Reply:
Done